sed/loader/flash/loader.py

```diff
@@ -299,6 +304,14 @@ def read_dataframe(
             ValueError: If neither 'runs' nor 'files'/'raw_dir' is provided.
             FileNotFoundError: If the conversion fails for some files or no data is available.
         """
+        detector = kwds.pop("detector", "")
+        force_recreate = kwds.pop("force_recreate", False)
+        processed_dir = kwds.pop("processed_dir", None)
+        debug = kwds.pop("debug", False)
+        remove_invalid_files = kwds.pop("remove_invalid_files", False)
+
+        if len(kwds) > 0:
+            raise ValueError(f"Unexpected keyword arguments: {kwds.keys()}")
         t0 = time.time()
```

Comment:
You are still passing kwds. to the metadata function! This won't work anymore.

Reply:
Thanks! Will fix.